examples/kafka/cluster.yaml

```diff
@@ -0,0 +1,23 @@
+# It can take approximately 15-25 minutes to create a cluster.
+apiVersion: kafka.aws.crossplane.io/v1alpha1
+kind: Cluster
+metadata:
+  name: kafka-test-server
+spec:
+  forProvider:
+    brokerNodeGroupInfo:
+      clientSubnets:
```

Comment:
@haarchri I realized after merge that we can reference subnets in this resource, could you make sure to follow up and add reference fields to `Subnet` for that?

Reply:
@hasheddan added via https://github.com/crossplane/provider-aws/pull/762 - if you have time would be cool to get a review